README.md

```diff
@@ -536,7 +536,13 @@ format:
 
   # all syft-json format options
   json:
-      
+    
+    # don't include space indention and newlines (same as jq -c)
+    compact: false
+    
+    # escape causes '<', '>', and '&' to be escaped in output
```

Comment:
Escaped how?

Reply:
defunct -- I'll remove the escape HTML feature (but leave the tests mostly intact)